aframe/tasks/infer/infer.py

```diff
@@ -125,8 +126,15 @@ def get_ip_address() -> str:
         Get the local, cluster-internal IP address
         Currently not a general function.
         """
-        nets = psutil.net_if_addrs()
-        return nets["enp1s0f0"][0].address
+
+        for _, addrs in psutil.net_if_addrs().items():
```

Comment:
Hell yeah, great to see this change. Have you tested this at all in different places? Would be good to make sure this works on both LLO and LHO, as well as on the deepclean nodes

Reply:
I ran on the deepclean nodes. Let me test out the dgx boxes. 

(you can thank GPT4o for this one)

Reply:
Can confirm works on LHO dgx1